ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml

```diff
@@ -1,4 +1,98 @@
setting date on host\"\nsudo date --set \\$future\nsleep 10\ndate\nfor i in {0..4}; do\n  ip=\"\\${ipList[i]}\"\n  ssh core@\\$ip \"sudo systemctl start crio && sudo systemctl start kubelet && date\"\ndone\necho \"waiting 120 seconds for csr creation\"\nsleep 120\nnodesReady=0\nretries=0\nset +euo pipefail\nwhile [ \\$nodesReady -ne 5 ] && [ \\$retries -lt 60 ]; do\n  pendingCSRs=\\$(oc get csr | grep Pending | wc -l)\n  if [ \\$pendingCSRs -ne 0 ]; then\n    echo \"Approving pending csrs\"\n    oc get csr -o name | xargs oc adm certificate approve\n  fi  \n  sleep 10\n  getNodes=\\$(oc get nodes | grep Ready | grep -v NotReady | awk '{ print \\$1 }')\n  nodesList=(\\$getNodes)\n  nodesReady=\\${#nodesList[@]}\n  if [ \\$nodesReady -eq 5 ]; then\n    echo \"All nodes Ready\"\n      fi\n  (( retries++ ))\n  done\nif [ \\$nodesReady -ne 5 ]; then\n  echo \"some nodes NotReady\"\n  oc get nodes\n  exit 1\nfi\npendingPods=2\nretries=0\nwhile [ \\$pendingPods -ne 0 ] && [ \\$retries -lt 120 ]; do\n  sleep 10\n  pendingPods=\\$(oc get pods -A --no-headers | grep -v -e \"Running\" -e \"Completed\" -e \"Terminating\" | wc -l)\n  if [ \\$pendingPods -eq 0 ]; then\n    echo \"All nodes Ready and no pending pods\"\n     fi\n  (( retries++ ))\n  done\nif [ \\$pendingPods -ne 0 ]; then\n  echo \"The following pods are Pending\"\n  oc get pods -A | grep -v -e \"Running\" -e \"Completed\" -e \"Terminating\"\n  exit 1\nfi\necho \"checking that all ClusterOperators are Degraded=False\"\ndegradedCOs=1\nretries=0\nwhile [ \\$degradedCOs -ne 0 ] && [ \\$retries -lt 120 ]; do\n  sleep 10\n  degradedCOs=\\$(oc get co --no-headers | awk '{ print \\$5 }' | grep -v \"DEGRADED\" | grep \"True\" | wc -l)\n  if [ \\$degradedCOs -eq 0 ]; then\n    echo \"All ClusterOperators Degraded=False\"\n      fi\n  (( retries++ ))\n  done\nif [ \\$degradedCOs -ne 0 ]; then\n  echo \"Some ClusterOperators are Degraded=True\"\n  oc get co\n  exit 1\nfi\necho \"checking that all ClusterOperators are Available=True\"\nunavailableCOs=1\nretries=0\nwhile [ \\$unavailableCOs -ne 0 ] && [ \\$retries -lt 120 ]; do\n  sleep 10\n  unavailableCOs=\\$(oc get co | awk '{ print \\$3 }' | grep -v \"AVAILABLE\" | grep \"False\" | wc -l)\n  if [ \\$unavailableCOs -eq 0 ]; then\n    echo \"All ClusterOperators Available=True\"\n      fi\n  (( retries++ ))\n  done\nif [ \\$unavailableCOs -ne 0 ]; then\n  echo \"Some ClusterOperators are Available=False\"\n  oc get co\n  exit 1\nfi\necho \"checking all ClusterOperators are Progressing=False\"\nprogressingCOs=1\nretries=0\nwhile [ \\$progressingCOs -ne 0 ] && [ \\$retries -lt 120 ]; do\n  sleep 10\n  progressingCOs=\\$(oc get co | awk '{ print \\$4 }' | grep -v \"PROGRESSING\" | grep \"True\" | wc -l)\n  if [ \\$progressingCOs -eq 0 ]; then\n    echo \"All ClusterOperators Progressing=False\"\n      fi\n  (( retries++ ))\n  done\nif [ \\$progressingCOs -ne 0 ]; then\n  echo \"Some ClusterOperators are Progressing=True\"\n  oc get co\n  exit 1\nfi\nexit 0\n"
```

Comment:
the inline syntax with `\n` is hard to review, why not this? 
```
value: |
  your code
  next line
```

Reply:
it was multi-line, but then when I `make jobs` it becomes 1 line :shrug:  I know, super hard to read, lemme ask platform team: @petr-muller how do i keep this multi-line? thanks

Reply:
I don't know if there is some form of input that the YAML marshaler would respect and output reasonable YAML on write. Not that I know of.

Reply:
thanks @petr-muller I'm closing this, moving to step-registry :) where it's more sane https://github.com/openshift/release/pull/12878